units/quotaon-root.service.in

```diff
@@ -0,0 +1,25 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Enable File System Quotas
+Documentation=man:quotaon(8)
+
+ConditionPathExists={{QUOTAON}}
```

Comment:
why this check?

if people ask for quota we should give them quota or fail. With this you will hide any failure in case people set the knob but have no quota tools enabled.

Reply:
(i see this is a pre-existing thing. would still prefer if this is dropped)